test/xbgpu/test_engine.py

```diff
@@ -490,29 +549,39 @@ def verify_beam_sensors(
         assert saturated_low[i] <= stream_diff.diff("output_b_clipped_samples_total") <= saturated_high[i]
 
         # Check that sensor value matches Prometheus
+        # NOTE: Verifying the timestamp on saturation count updates is not as predictable
+        # as saturation relies on dithering. As a result, the timestamp field here is mocked
+        # out, and other sensor updates are verified more completely.
```

Comment:
I wouldn't use the term "mocked out" here - that normally implies replacing functional behaviour. I'd say the timestamp field is "ignored".